src/keplr/index.ts

```diff
@@ -15,45 +20,37 @@ export function getChains() {
   };
 }
 
-export function getSigner(chain: any) {
-  return (window as any).getOfflineSignerAuto(chain.id);
+export function getSigner(chain: Chain) {
+  return window.getOfflineSignerAuto(chain.id);
 }
 
-export async function get(chain: any, signer: any, endPoint: string) {
-  const customAminoTypes: any = new AminoTypes({
+export async function get(chain: Chain, signer: OfflineSigner | OfflineDirectSigner, endPoint: string) {
+  const customAminoTypes = new AminoTypes({
     "/akash.cert.v1beta2.MsgCreateCertificate": {
       aminoType: "cert/cert-create-certificate",
-      toAmino: ({ owner, cert, pubkey }: any) => {
-        const buf: any = Certificate.encode(
+      toAmino: ({ cert, pubkey }: MsgCreateCertificate) => {
+        const buf = Certificate.encode(
           Certificate.fromPartial({
-            owner,
             cert,
             pubkey
-          } as any)
+          })
         ).finish();
         const encoded = Buffer.from(buf);
         return encoded.toString("base64");
       },
-      fromAmino: ({ owner, cert, pubkey }: any) => {
+      fromAmino: ({ cert, pubkey }: MsgCreateCertificate) => {
         return Certificate.fromPartial({
-          owner,
           cert,
           pubkey
-        } as any);
+        });
       }
     }
-  } as any);
+  });
 
   const myRegistry = new Registry([...defaultRegistryTypes, ...getAkashTypeRegistry()]);
 
   return await SigningStargateClient.connectWithSigner(endPoint, signer, {
-    bip44: {
```

Comment:
same question on https://github.com/akash-network/akashjs/pull/78/files#diff-108edb79510ba985a92b042dc6c131c0f364df49148cab4252b5168453e3c31eR55
bip44 is not a known prop so it actually is not used most prob